theme/scripts/cart.ts

```diff
@@ -285,9 +285,10 @@ export default(function() {
       })
```

Comment:
Line 270 above: 

I hadn't ever done this: `[].slice.call(...)` - it looks like it's just to convert a NodeList into an array? You can use `Array.from(document.querySelectorAll(...))`. Or better yet just make a utility function:

```
function getElements(selector: string): Element[] {
  return Array.from(document.querySelectorAll(selector))
} 
```

Then your usage of it will be a little more semantic